literalai/api/dataset_helpers.py

```diff
@@ -98,13 +99,13 @@ def create_experiment_helper(
     api: "LiteralAPI",
     name: str,
     dataset_id: Optional[str] = None,
-    prompt_id: Optional[str] = None,
+    prompt_experiment_id: Optional[str] = None,
```

Comment:
I'd say we call it `prompt_variant_id`. We should rename it in the platform code too, else it's going to lead to errors.